controllers/clusterversion_controller.go

```diff
@@ -55,6 +55,8 @@ const (
 	// ClusterVersionName is the name of the ClusterVersion object in the
 	// openshift cluster.
 	clusterVersionName = "version"
+
+	ClusterVersionFinalizer = "ocs-client-operator.ocs.openshift.io/ocspvcused"
```

Comment:
i think `ocspvcused` is the right one to use here, we can have no PVC but we can have volumesnapshots and also we are not checking for any resources before deletion

Reply:
@Madhu-1 I believe the resource checking is coming in a different PR (https://github.com/red-hat-storage/ocs-client-operator/pull/81)

But I agree about the naming for the finalizer:
@bernerhat can we change i to ocs-client-operator.ocs.openshift.io/storageinuse" (Storage in Use)
@Madhu-1 if you have a suggestion for a different name please reply here

Reply:
`ocs-client-operator.ocs.openshift.io/storageinuse` make sense 👍🏻